### PR TITLE
Revup of CLI classes to data model

### PR DESCRIPTION
**Pull request by gsvarovsky**
Example data (in playground) for a single entry & session:

```
[
  {
    "@id": "3uyGG8jtvPEM46fndQ7TxS",
    "@type": "http://timeld.org/#Session",
    "http://timeld.org/#start": {
      "@value": "2022-05-06T15:59:42.070Z",
      "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
    }
  },
  {
    "@id": "3uyGG8jtvPEM46fndQ7TxS/1",
    "@type": "http://timeld.org/#Entry",
    "http://timeld.org/#activity": "testing",
    "http://timeld.org/#duration": 60,
    "http://timeld.org/#session": {
      "@id": "3uyGG8jtvPEM46fndQ7TxS"
    },
    "http://timeld.org/#start": {
      "@value": "2022-05-06T15:59:49.787Z",
      "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
    },
    "https://w3id.org/valueflows#provider": {
      "@id": "https://www.linkedin.com/in/gsvarovsky"
    }
  }
]
```

### CODE REVIEW

File: architecture/data-model/class-data-model.puml [hunk 1]
```diff
@@ -1,19 +1,27 @@
 @startuml
 'https://plantuml.com/class-diagram
 
+hide circle
+
 package ts {
     class Timesheet {
-        entry : Entry
```
Comment (gsvarovsky) [inline architecture/data-model/class-data-model.puml:6]: Entry points to timesheet, so removed from here

File: architecture/data-model/class-data-model.puml [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
         project : Project [*]
     }
 
+    class Session {
+        start : xsd:dateTime
+        timesheet : Timesheet
+    }
+
+    Session "1" *-- "*" Entry
+
```
Comment (gsvarovsky) [inline architecture/data-model/class-data-model.puml:17]: Session object represents a single session and is used to scope entries

File: architecture/data-model/class-data-model.puml [hunk 3]
```diff
@@ -1,19 +1,27 @@
...
     class Entry {
+        session : Session
         activity : xsd:string
         hourlyRate : hourlyRate
-        contributor : vf:Provider
-        startTime : xsd:dateTime
-        duration: om2:Duration
-        timesheet : Timesheet
+        vf:provider : foaf:Agent
```
Comment (gsvarovsky) [inline architecture/data-model/class-data-model.puml:22]: `vf:Provider` does not exist as a class. So I have used the `vf:property` property to link direct to a `foaf:Agent`

File: architecture/data-model/class-data-model.puml [hunk 4]
```diff
@@ -1,19 +1,27 @@
...
+        start : xsd:dateTime
+        duration : xsd:decimal
```
Comment (gsvarovsky) [inline architecture/data-model/class-data-model.puml:24]: Duration is number of minutes, as agreed

File: architecture/data-model/class-data-model.puml [hunk 5]
```diff
@@ -40,19 +51,16 @@ package ts {
...
-        name : xsd:string
+        foaf:name : xsd:string
     }
 
     Project "*" -- "0..1" Client
 
-    note as N1
-    Project's client property is also
+    note right of Project::client
+    client property is also
```
Comment (gsvarovsky) [inline architecture/data-model/class-data-model.puml:60]: Switched to using a comment on the `client` property

**Comment by Angus-McAllister** [comment]
Reviewed during in-person meeting, with corresponding discussion, and changes agreed.

**Review by Angus-McAllister — COMMENTED** [review]
Reviewed during in-person meeting, and agreed.

**Pull request merged**